examples/showcase/basic-scene/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Basic Scene with Environment - A-Frame</title>
+    <meta name="description" content="Basic Scene with Environment - A-Frame">
+    <script src="https://aframe.io/releases/1.4.0/aframe.min.js"></script>
```

Comment:
You need to use `../../../dist/aframe-master.js` here like the other showcases.

Reply:
If you use a versioned url here for education purpose so that the user can copy paste the code, that's another story but be aware that currently the `npm run prerelease` command that use the `scripts/release.js` script only update the version in `*.md` files, not in `*.html` files.

Reply:
Ok, hadn't appreciated that limitation.

I'll point to master and add a comment explaining what to do if you want to run the code in a different context.

Reply:
Yes. It's a tradeoff. Also as the release manager, before a release I want all the examples to use latest aframe-master to test for any regression.